# joint name/id maps
JointNameMap joint_map

# group name
string group_name

# group ID
int8 group_id

# group control frame
string control_frame

# group planning frame
string planning_frame

# joint mask
JointMask joint_mask